boards/arm/mimxrt1040_evk/doc/index.rst

```diff
@@ -291,6 +291,13 @@ should see the following message in the terminal:
 Troubleshooting
 ===============
 
+USER_LED D8
+-----------
+The MIMXRT1040-EVK board ships with the wireless module in the M.2 connector,
+and with jumper J80 shorted.  This causes a conflict with the USER_LED D8,
+and the LED will not turn off.  When using USER_LED D8, remove J80, like when
+using the `blinky` sample.
```

Comment:
The wording on the last sentence is a bit confusing. Suggestion:

"Samples and applications using USER_LED D8, like `blinky`,  will require removal of J80 jumper.

Reply:
Thanks, @dleach02 .  I made that change